src/sdc11073/xml_types/actions.py

```diff
@@ -0,0 +1,56 @@
+from ..namespaces import default_ns_helper as ns_hlp
+
+_ActionsNamespace = ns_hlp.SDC.namespace
+
+
+class Actions:

```

Comment:
make enum (and according to pep make attributes `snake_case`)

Reply:
enum done, but changing the member names is a massive change, not in this PR.